### ISSUE TITLE
Butchering reports of misleading possible CBM on every corpse.

### ISSUE THREAD

**Issue by jim-huynh** [CleverRaven/Cataclysm-DDA#24876]
Version: 0.c-30616-g5d18af2

Every butchering event shows a "Your butcher tool encounter something implanted in this corpse, but your rough cut destroys it." or some variant of a possible CBM regardless of there being a possible CBM.

Since this is the PC doing something, he shouldn't report to himself that there was something implanted when there wasn't.  Assuming that the PC completely takes apart a corpse, they would know after the fact that there was, or was not something implanted in the corpse.

If there was indeed a CBM implanted, then a butchering event that destroyed a CBM should yield a destroyed CBM item, and not just bypass it.

**Comment by FulcrumA** [comment]
Can confirm in bizarre situations such as this:
![uhoh](https://user-images.githubusercontent.com/38487938/44222459-d7053d80-a184-11e8-8211-27cd8a43e250.png)
On top of that, said coyote provided me with 28 pieces of meat which is quite excessive, at least at lower survival levels without REALLY good butchering tool.

**Comment by jim-huynh** [comment]
Yeah he multiplied get_meat_chunks() by some rather large numbers, and I don't see anything in the new code that reduces it back to the old yields after processing.

**Comment by nexusmrsep** [comment]
It's not that its excessive, its just based on weight, and guess what? Coyote as SMALL weights 40 kg, so the yield is a derivative from that. Yeah, that's a monster coyote...

**Comment by jim-huynh** [comment]
You're right, a 40 kg animal should probably yield around 30-40% of it's weight in meat.  At 0.2 kg per meat object, your numbers are closer to expected, but it does change survival balance.

Things like a full sized deer (large) is going to be a jackpot.

**Comment by nexusmrsep** [comment]
There should be some tweaks for the weight of small animals followed by yields.
